.travis.yml

```diff
@@ -12,11 +12,24 @@ notifications:
     recipients:
       - 130s@2000.jukuin.keio.ac.jp
 env:
+  global:
+    - MOVEIT_CI_TRAVIS_TIMEOUT=85  # Travis grants us 90 min, but we add a safety margin of 5 min
+    - ROS_DISTRO=melodic
+    - ROS_REPO=ros
+    - UPSTREAM_WORKSPACE=moveit.rosinstall
+    # moveit_ros_perception: mesh_filter_test fails due to broken Mesa OpenGL
+    # moveit_experimental: test succeeds locally, not on Travis
+    - TEST_BLACKLIST="moveit_ros_perception moveit_experimental"
+    - CXXFLAGS="-Wall -Wextra -Wwrite-strings -Wunreachable-code -Wpointer-arith -Wredundant-decls -Wno-unused-parameter -Wno-unused-but-set-parameter -Wno-unused-function"
+    - WARNINGS_OK=false
   matrix:
-    - ROS_DISTRO=kinetic ROS_REPO=ros              TEST=clang-format
-    - ROS_DISTRO=melodic ROS_REPO=ros              UPSTREAM_WORKSPACE=moveit.rosinstall
-    - ROS_DISTRO=melodic ROS_REPO=ros-shadow-fixed UPSTREAM_WORKSPACE=moveit.rosinstall
+    - TEST="clang-format catkin_lint"
+    - TEST=clang-tidy-fix
+    - ROS_DISTRO=melodic
+    - ROS_DISTRO=kinetic
```

Comment:
Should we still test this with kinetic?

Reply:
the moveit repo still builds with kinetic, so i think this should too